float64 yawAngle
int32   leftEncoder
int32   rightEncoder
